rwatch/ui/layer/action_bar_layer.c

```diff
@@ -29,7 +29,7 @@ ActionBarLayer *action_bar_layer_create()
 
 void action_bar_layer_destroy(ActionBarLayer *action_bar)
 {
-    layer_destroy(action_bar);
+    layer_destroy(action_bar_layer_get_layer(action_bar));
```

Comment:
(without seeing code)  surely action_bar->layer is the right semantic here

Reply:
It's the same proper way apps would get access to the layer, thought it should apply here as well.

In case in the future the struct changes for some reason.